prow/version/metrics.go

```diff
@@ -30,6 +32,12 @@ var (
 
 func init() {
 	prometheus.MustRegister(prowVersion)
+	// For components that import current package, if `GatherProwVersion` if not
+	// explicitly called, there will be no metrics for `prow_version`, and when
+	// querying prometheus for `prow_version` the value will be zero, which
+	// is inaccurate. Since the version would not change for the running binary,
+	//  doing this once when binary starts should fix this problem.
+	GatherProwVersion(logrus.WithContext(context.Background()))
```

Comment:
In this case we can avoid exposing the `GatherProwVersion` function entirely and only use it here, right?

Reply:
good point, unexposed it and removed all references from elsewhere